PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.cpp

```diff
@@ -89,16 +89,17 @@ void Version::setVersionFrom(generic_string filePath)
 		unsigned char* buffer = new unsigned char[bufferSize];
 		::GetFileVersionInfo(filePath.c_str(), handle, bufferSize, buffer);
 
-		VS_FIXEDFILEINFO* lpFileInfo;
+		VS_FIXEDFILEINFO* lpFileInfo = nullptr;
 		UINT cbFileInfo = 0;
-		VerQueryValue(buffer, TEXT("\\"), (LPVOID*)&lpFileInfo, &cbFileInfo);
+		VerQueryValue(buffer, TEXT("\\"), reinterpret_cast<LPVOID*>(&lpFileInfo), &cbFileInfo);
```

Comment:
Unfortunately, I could not test this method `void Version::setVersionFrom(generic_string filePath)`. But I feel, I should work perfectly.

Reply:
Testing of method `void Version::setVersionFrom(generic_string filePath)` is done now. It works fine.